### PR TITLE
[chromecast-caf-receiver] Update declarations according to SDK docs (3)

### PR DESCRIPTION
**Pull request by marcoreni**
Next step in bringing this package up to date with the [Web Receiver Documentation](https://developers.google.com/cast/docs/reference/web_receiver)

NOTES:

- 95% of the changes are docs cleanup / fixes. I tried adding a link to every class / enum, and fixing JSDoc issues
- This set of PRs is file-based, so that they're easier to review. This PR handles
  - `cast.framework.messages.d.ts`
- I followed Google Javascript Styleguide for comments.

Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: _see above_

### CODE REVIEW

File: types/chromecast-caf-receiver/cast.framework.messages.d.ts [hunk 1]
```diff
@@ -1100,6 +1128,11 @@ export class QueueItem {
      */
     playbackDuration?: number | undefined;
 
+    /**
+     * Used to track original order of an item in the queue to undo shuffle.
+     */
+    orderId?: number | undefined;
```
Comment (peterblazejewicz) [inline types/chromecast-caf-receiver/cast.framework.messages.d.ts:1134]: where does this one come from?:
https://developers.google.com/cast/docs/reference/web_sender/chrome.cast.media.QueueItem?hl=en

Comment (peterblazejewicz) [reply types/chromecast-caf-receiver/cast.framework.messages.d.ts:1134]: https://developers.google.com/cast/docs/reference/web_receiver/cast.framework.messages.QueueItem?hl=en#orderId
?

File: types/chromecast-caf-receiver/cast.framework.messages.d.ts [hunk 2]
```diff
@@ -2312,6 +2455,37 @@ export class Break {
     position: number;
 }
 
+/**
+ * Content rating (parental rating, maturity rating).
+ * @see https://developers.google.com/cast/docs/reference/web_receiver/cast.framework.messages.ContentRating
+ */
+export class ContentRating {
+    constructor();
+
```
Comment (peterblazejewicz) [inline types/chromecast-caf-receiver/cast.framework.messages.d.ts:2464]:
```suggestion
```
this serves no purpose

File: types/chromecast-caf-receiver/cast.framework.messages.d.ts [hunk 3]
```diff
@@ -2381,3 +2554,32 @@ export class AudiobookContainerMetadata {
...
     releaseDate?: string | undefined;
 }
+
+/**
+ * Tracks information.
+ * @see https://developers.google.com/cast/docs/reference/web_receiver/cast.framework.messages.TracksInfo
+ */
+export class TracksInfo {
+    constructor();
+
```
Comment (peterblazejewicz) [inline types/chromecast-caf-receiver/cast.framework.messages.d.ts:2564]:
```suggestion
```
serves no purpose

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@marcoreni thanks!

File: types/chromecast-caf-receiver/cast.framework.messages.d.ts (hunk 1 above)
Comment (marcoreni) [reply types/chromecast-caf-receiver/cast.framework.messages.d.ts:1134]: The first link you sent is related to a different package, `chromecast-caf-sender` . The correct reference is the second link, which contains the attribute.

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@marcoreni thx!

**Comment by marcoreni** [comment]
Ready to merge

**Pull request merged**